source/includes/etl_reports/_etl_lead_attribution.md

```diff
@@ -1,17 +1,36 @@
-### ETL Lead Attribution Report
+### **ETL Lead Attribution Report**
+<a name="Lead Attribution report"></a>
 
-#### Report Params
+### Resource Overview&nbsp;
 
-When creating an ETL Lead Attribution Report, provide the following fields in `report_params`:
-
-| Param | Function |
+|Method|URI Format|
 |---|---|
-|`gmaid`|Global Master Advertiser ID.  Only events for a single advertiser can be returned in a single report|
-|`start_date`|Restricts the results to those occurring on or after this date.|
-|`end_date`|Restricts the results to those occurring on or before this date.|
+|GET|/client_reports/unified_display/[gmaid]?[query_params]|When creating an ETL Lead Attribution Report, provide the following fields in `report_params`:
```

Comment:
To create an ETL report, a POST is required.  This URL is also incorrect.

I also feel that the fact that there is an asynchronous request/response process used to generate this report isn't clear at all in this documentation (this is more of a comment on earlier work).  The documentation must stress how you create the job and then need to poll an API endpoint until the job is complete.  No mention of the GET portion of the request to retrieve the status and the actual file is made here.

Reply:
I will rewrite it.